### ISSUE TITLE
Collapsable checklist on card

### ISSUE THREAD

**Issue by czinkos** [wekan/wekan#1591]
It would be great to have collapsable checklists on cards.

**Comment by MikeRatcliffe** [comment]
Yes, having checklists visible on cards would be amazing so +1 from me!

**Comment by JannetGen** [comment]
This would be a really nice quality-of-life improvement. Long checklists can make a card pretty cluttered, so being able to collapse them and expand only when needed would keep the board much cleaner.
